types/node/globals.d.ts

```diff
@@ -53,10 +53,34 @@ declare global {
     // Same as module.exports
     var exports: any;
 
+    interface GCFunction {
+        (options: {
+            execution?: "sync";
+            flavor?: "regular" | "last-resort";
+            type?: "major-snapshot" | "major" | "minor";
+            filename?: string;
+        }): void;
+        (options: {
+            execution?: "async";
```

Comment:
```suggestion
            execution: "async";
```
so as not to overlap with the sync signature.